app/komponenter/veiledning/veiledning.module.css

```diff
@@ -0,0 +1,8 @@
+.veilederPanel {
+  width: 100%;
+  padding: 0;
+
+  @media (max-width: 768px) {
```

Comment:
Her ville jeg lurt på hvorfor akkurat 768px. Dette kan man gjøre mer lesbart ved å dra den ut i en global css variabel, og kalle den `mobil` eller `tablet` eller hva enn media med max-width på 768px er 😊

Reply:
Den kommer fra [brekkepunkter på aksel!](https://aksel.nav.no/grunnleggende/styling/brekkpunkter).  Den ser kanskje litt tilfeldig ut, men den er valgt fra aksel sin dokumentasjon ☀️
Vi kan sikkert gjer den global ja!

Reply:
I den ytterste utils-mappen kan man opprette en fil ved navn constants.ts eller konstanter.ts som exporterer følgende;

`
export const device = {
  tablet: '768px';
  }
  
  også benytte device.tablet istedet for 768px for eksempel?